momento-sdk/src/main/java/momento/sdk/CacheClient.java

```diff
@@ -262,6 +264,18 @@ public CompletableFuture<GetResponse> get(String cacheName, String key) {
     return scsDataClient.get(cacheName, key);
   }
 
+  /**
+   * Get the cache values stored for the given keys.
+   *
+   * @param cacheName Name of the cache to get the item from.
+   * @param keys The keys to get.
+   * @return Future with {@link GetBatchResponse} containing the status of the get batch operation
+   *     and the associated value data.
+   */
+  public CompletableFuture<GetBatchResponse> getBatch(String cacheName, Iterable<String> keys) {
```

Comment:
The gRPC calls for get batch and set batch are blocking, so the user-facing calls are only pretend async. I did that to keep the API consistent, but I can change it if we want to ensure users know these calls will block.